docs/domains/Domain.json

```diff
@@ -354,6 +359,11 @@
           "type": "number",
           "minimum": 0
         },
+        "maxClusterConcurrentShutdown": {
+          "description": "The maximum number of cluster member Managed Server instances that the operator will shutdown in parallel for a given cluster, if `maxConcurrentShutdown` is not specified for a specific cluster under the `clusters` field. A value of 0 means there is no configured limit. Defaults to 1.",
```

Comment:
How about:

The default maximum WebLogic Server instances that a cluster will shutdown in parallel when it is being partially shutdown by lowering its replica count. You can override this default on a per cluster basis by setting the cluster's `maxConcurrentShutdown` attribute. A value of 0 means there is no limit. Defaults to 1.

Reply:
I have made changes in DomainSpec.java to use above wordings. Please let me know if I missed something. Thanks.